src/sentry/demo/settings.py

```diff
@@ -0,0 +1,18 @@
+from datetime import timedelta
+from django.conf import settings
+
+from sentry.conf.server import *  # NOQA
+
+"""
+To get this file to load, add the follwing to your sentry.conf.py file:
+
+from sentry.demo.settings import *
+
+"""
+
+if settings.DEMO_MODE:
+    settings.CELERYBEAT_SCHEDULE["demo_delete_users_orgs"] = {
```

Comment:
Remove `settings.`

Reply:
@wedamija the problem with that is it causes flake8 to fail:
![Screen Shot 2021-03-05 at 9 49 43 AM](https://user-images.githubusercontent.com/8533851/110153606-22f64d00-7d98-11eb-8454-48dbad5c9e05.png)

It fails even if I try to set default values for them:

![Screen Shot 2021-03-05 at 9 50 28 AM](https://user-images.githubusercontent.com/8533851/110153686-3e615800-7d98-11eb-840e-1af59113676f.png)

What do you think is the best solution to this?

Reply:
Kind of confused about this, seems to work ok in getsentry: https://github.com/getsentry/getsentry/blob/f4dffb91b875942976cae7f882eb76565984d88b/getsentry/conf/settings/defaults.py#L451

Is it because we have the `#NOQA` on the * import? Or wondering if we just skip config files in flake8 usually?

Reply:
@wedamija It's cause we have this at the top:
```
# flake8: noqa
```
Flake8 just ignores the file 🙃